libs/dh/admin/feature-user-management/src/lib/shared/dh-user-roles.component/dh-user-roles.component.html

```diff
@@ -30,22 +30,43 @@
           *ngTemplateOutlet="
             userRolesHeadline;
             context: {
-              $implicit: numberOfRoles$ | push
+              $implicit:
+                (selectMode ? numberOfAssignableRoles$ : numberOfSelectedRoles$)
+                | push
             }
           "
         ></ng-container>
       </watt-card-title>
-      <div class="userRoles" *ngFor="let org of userRoleView.organizations">
+      <div
+        class="userRoles"
+        [ngClass]="{ selectMode: selectMode }"
+        *ngFor="let org of userRoleView.organizations"
+      >
         <ng-container *ngFor="let actor of org.actors">
           <h6>
             {{ org?.name }} | GLN {{ actor.actorNumber }} |
-            {{ actor.userRoles | joinUserRoles }}
+            {{ actor.actorMarketRoles | joinMarketRoles }}
```

Comment:
suggestion: maybe we can fallback to an em dash (using the `dhEmDashFallback` pipe).

question: Can `actorMarketRoles` be an empty array?

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1374/commits/3f88cdb5ead6f7ee96b0ce84b0a4eff8bb5facc7